x-pack/test/cases_api_integration/security_and_spaces/tests/common/internal/get_attachment_stats.ts

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { CaseResponse } from '@kbn/cases-plugin/common';
+import {
+  globalRead,
+  noKibanaPrivileges,
+  obsOnly,
+  obsOnlyRead,
+  obsSec,
+  obsSecRead,
+  secOnly,
+  secOnlyRead,
+  superUser,
+} from '../../../../common/lib/authentication/users';
+import { FtrProviderContext } from '../../../../common/ftr_provider_context';
+import {
+  bulkCreateAttachments,
+  createCase,
+  deleteAllCaseItems,
+  getAttachmentStats,
+} from '../../../../common/lib/api';
+import {
+  getFilesAttachmentReq,
+  postCaseReq,
+  postExternalReferenceSOReq,
+} from '../../../../common/lib/mock';
+
+// eslint-disable-next-line import/no-default-export
+export default ({ getService }: FtrProviderContext): void => {
+  const supertest = getService('supertest');
+  const es = getService('es');
+
+  describe('get_attachment_stats', () => {
+    describe('delete all cases entities after each', () => {
+      afterEach(async () => {
+        await deleteAllCaseItems(es);
+      });
+
+      describe('response structure', () => {
+        it('should return the correctly formatted response', async () => {
+          const postedCase = await createCase(supertest, postCaseReq);
+          const stats = await getAttachmentStats({ supertest, caseId: postedCase.id });
+
+          expect(stats).to.eql({
```

Comment:
Just to ensure we don't add any other fields at the moment. If we add more stats in the future we'll need to update this accordingly.